tests/helpers/cluster.ts

```diff
@@ -140,28 +146,36 @@ function restart(cluster: NatsServer[]) {
 }
 
 function waitForStop(): void {
+  const isWindows = Deno.build.os === "windows";
+  const isLinux = Deno.build.os === "linux";
```

Comment:
lets just test for windows, the rest is assumed to be some sort of unix, otherwise we'll fail and add more complexity for other things that will run

Reply:
This is only for signal handling on Linux swapping `ctrl-T` with `ctrl-Z` since `SIGINFO` doesn't work on Linux. It'd be nice to have the option to run on Linux as well. BTW macOS should not be affected.